app/assets/stylesheets/custom.scss

```diff
@@ -128,12 +127,35 @@ a {
 
 #store-name {
   color: #504d4f;
+  float: left;
+  font-size: 1.6em;
+  margin: auto;
 }
 
 .retire-btn {
   margin-bottom: 54px;
 }
 
+#complete-your-loan {
+  text-align: center;
+}
+
+#profile-title {
+  text-shadow: 0 2px 3px #666;
+}
+
+#profile-data {
+  background-color: #e3e3e3;
+  border: 1px solid #e3e3e3;
+  border-color: #d7d7d7;
+  border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
+  box-shadow: inset 0 1px 1px rgba(0,0,0,.05);
```

Comment:
`.05` should be written with a leading zero as `0.05`<br>Commas in function arguments should be followed by a single space